master/internal/telemetry/reports.go

```diff
@@ -115,7 +99,7 @@ func ReportAllocationTerminal(
 		"allocation_id": a.AllocationID,
 		"task_id":       a.TaskID,
 		"start_time":    a.StartTime,
-		"end_time":      *a.EndTime,
```

Comment:
tbh I don't know why this was a pointer in the first place, doesn't need to be per struct defn